from core.exceptions.base import BaseException

# DNS Custom Exceptions
